library/markdown.php

```diff
@@ -15,7 +15,9 @@ function Markdown($text) {
 	$stamp1 = microtime(true);
 
 	# Read file and pass content through the Markdown parser
```

Comment:
It is used in the AppNet addon. There's also the timestamp recording for the time spent by the Markdown transformer that would be lost in the deletion of this file. I think it's going to stay there until we move more stuff around to be auto-loaded.